datafusion/core/tests/dataframe/mod.rs

```diff
@@ -1121,6 +1121,39 @@ async fn join() -> Result<()> {
     Ok(())
 }
 
+#[tokio::test]
+async fn join_coercion_unnnamed() -> Result<()> {
```

Comment:
Should "unnnamed" be corrected to "unnamed"?

Reply:
Yes, thank you. That is a typo

Reply:
```suggestion
async fn join_coercion_unnamed() -> Result<()> {
```

Reply:
- in https://github.com/apache/datafusion/pull/14403